v2/tools/generator/internal/astmodel/known_locals_set.go

```diff
@@ -28,15 +28,44 @@ func NewKnownLocalsSet(idFactory IdentifierFactory) *KnownLocalsSet {
 	}
 }
 
+// flectOverrides is an ordered list of overrides for flect pluralization
+// Earlier overrides take precedence over later ones
+var flectOverrides = []struct {
```

Comment:
Just for your information, Flect supports user data. If you add your own set of "single":"plural" data (`inflections.json`) in the app's working directory, the JSON file will be parsed and override the Flect defaults. It could be useful.

For example:

```json
{
  "FIPS": "FIPS",
  "Id": "Ids",
  "Ip": "Ips",
  "as": "as"
}
```